tests/benchdnn/dnnl_memory.cpp

```diff
@@ -900,7 +913,16 @@ int dnn_mem_t::initialize(
     SAFE(initialize_memory_create(handle_info), CRIT);
 
     if (handle_info.is_allocate()) {
-        if (!has_bench_mode_modifier(mode_modifier_t::no_ref_memory)) map();
+        // Indirectly accessed memory objects can't have random data in them
```

Comment:
I know what the problem is but I fail to comprehend the comment :) 
Does it say that the metadata buffers cannot be filled with random values because it will result in a crash during decoding the tensor?

Reply:
Thanks for pointing to this. I've updated the comment, please check it out.